### ISSUE TITLE
Latex formulas not showing in tutorials HTML pages

### ISSUE THREAD

**Issue by VictorLamoine** [PointCloudLibrary/pcl#1307]
Reported on [pcl-users](http://www.pcl-users.org/Cannot-see-math-equations-in-tutorial-pages-td4039233.html)

I think this is due to the new Travis script, a package must be missing! From what I remember I had to install theses to get the Latex formulas working:
```
$ sudo apt-get install doxygen graphviz sphinx3 python-pip
$ sudo pip install sphinxcontrib-doxylink
```

**Comment by taketwo** [comment]
We have now:

 * `doxygen-latex`, installed from `apt`
 * `doxygen 1.8.9.1`, installed from source
 * `sphinxcontrib-doxylink`, installed from `pip`

Any idea what specifically we are missing?

**Comment by jspricke** [comment]
I did a diff, and found at least dvipng, can someone test if that is enough?

**Comment by taketwo** [comment]
Ok, I'll test.

**Comment by taketwo** [comment]
Fixed (#1308)
